src/content-blocks/BlockKlaar/BlockKlaar.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FunctionComponent } from 'react';
+
+import classnames from 'classnames';
+
+import { DefaultProps } from '../../types';
+
+import './BlockKlaar.scss';
+
+export interface BlockKlaarProps extends DefaultProps {
+	date: string;
```

Comment:
zou dit niet beter een date iso string zijn?
Aangezien we deze date gaan gebruiken om de UUID van de klaar json te bepalen. Een iso string is veel makkelijker te parsen dan een manueel geformatteerde string en dan moet de date picker in de client niet beslissen hoe dit gepresenteerd zal worden door de block (separation of concerns).